frontend/src/components/RoutePlanner/RoutePlanner.tsx

```diff
@@ -247,12 +226,13 @@ export default RoutePlanner;
 export function TripOptionsDisplay(props: {
```

Comment:
Why is here some duplicate with "TripOptionsDisplay" (separate file - which is not used at the moment)?

Reply:
I thing this is part of the refactor...